game/lib/journeys/journey_cell.dart

```diff
@@ -160,7 +159,6 @@ class JourneyPreview extends StatefulWidget {
   final String journeyName;
   final String description;
   final int numberCompleted;
-  final bool isCompleted;
   final String difficulty;
   final int points;
   final int challenge_points;
```

Comment:
camelCase, but also think about if you can reuse the challenge cell instead of making a whole journey cell